.gitignore

```diff
@@ -0,0 +1,199 @@
+# @see https://www.drupal.org/project/drupal/issues/3082958#comment-13433619
+# This file contains .gitignore rules that are often used with Drupal projects.
+# Because .gitignore is specific to your site and its deployment processes, you
+# may need to uncomment, add, or remove rules.
+#
+# To ignore all paths that composer manages when using this project template,
+# remove all double-hashed (##) lines.
+
+
+# Ignore configuration files that may contain sensitive information.
+#
+# Typically, settings.php and related files are not committed to the
+# repository because they contain information such as the database
+# credentials that could be used to compromise a site. Sometimes,
+# a settings.php might be committed to the repository if it reads
+# sensitive information from environment variables or other sources.
+# ------------------------------------------------------------------
+/web/sites/*/settings*.php
```

Comment:
In the last Drupal project I worked on (and which is therefore my reference point), we actually committed `settings.php` (cf. https://github.com/itk-dev/giv-din-stemme/blob/develop/.gitignore#L20).